src/Lexer.js

```diff
@@ -46,6 +46,7 @@ function Token(text, data, position) {
 var tokenRegex = new RegExp(
     "([ \r\n\t]+)|(" +                                // whitespace
     "---?" +                                          // special combinations
+    "|[\u0391-\u03A9\u03B1-\u03C9]" +                 // Greek letters
```

Comment:
`\u03D6` is `\varpi` ϖ, `\u03D5` is `\phi` ϕ, `\u03F5` is `\epsilon` ϵ. KaTeX outputs all these (try `\varpi \phi \epsilon` and look at the Unicode code points of the output); do you want to include them?